src/Slider/Slider.js

```diff
@@ -604,7 +620,7 @@ class Slider extends Component {
         <span>{description}</span>
         <span>{error}</span>
         <div
-          style={prepareStyles(sliderStyles)}
+          style={styles.root}
```

Comment:
Could we change the name of `root`? I think that we should keep it for our real *root*.